public/app/plugins/datasource/loki/LogContextProvider.ts

```diff
@@ -33,6 +33,26 @@ export class LogContextProvider {
     this.appliedContextFilters = [];
   }
 
+  getLogRowContextQuery = async (
```

Comment:
Considering this is almost identical code that we use in `getLogRowContext`, couldn't we re-use it?

Reply:
Fixed in [e8494cd](https://github.com/grafana/grafana/pull/66777/commits/e8494cdcbb3dba269582d3f4d9dafde6dd7df796)